src/getting-started/tooling/probe-rs.md

```diff
@@ -22,23 +28,31 @@ Espressif products containing the [`USB-JTAG-SERIAL` peripheral][usb-jtag-serial
 [rtt]: https://wiki.segger.com/RTT
 [prober-rs-installation]: https://probe.rs/docs/getting-started/installation/
 [prober-rs-setup]: https://probe.rs/docs/getting-started/probe-setup/
-[usb-jtag-serial]: index.md#usb-jtag-serial-peripheral
 
-## Flashing with `probe-rs`
+## Flashing an Application
+
+Assuming you have built an ELF binary by other means already, `probe-rs` can be used to download it to your device with:
 
-`probe-rs` can be used to flash applications to your target since it supports the [ESP-IDF image format][idf-image].
-  - Example command for flashing an ESP32-C3: `probe-rs run --chip esp32c3`
+
+```shell
+probe-rs run
+```
+
+## Using `probe-rs` as a Cargo Runner
 
 The flashing command can be set as a custom Cargo runner by adding the following to your project's `.cargo/config.toml` file:
 
 ```toml
 [target.'cfg(any(target_arch = "riscv32", target_arch = "xtensa"))']
-runner = "probe-rs run --chip esp32c3"
+runner = "probe-rs run"
 ```
 
 With this configuration, you can flash and monitor your application using `cargo run`.
 
-[idf-image]: https://docs.espressif.com/projects/esptool/en/latest/esp32c3/advanced-topics/firmware-image-format.html
+This is automatically done when the project is generated via [`esp-generate`][esp-generate].
+
+[esp-generate]: esp-generate.md
+
 
 ## VS Code Extension
```

Comment:
Shall I remove the debugging stuff? As there is an advanced/debugging chapter

Reply:
Probably yeah. I think this is documented by probe-rs already too?